src/components/timeline/Row.svelte

```diff
@@ -207,7 +209,13 @@
 <div class="row-root" class:active-row={$selectedRow ? $selectedRow.id === id : false}>
   <!-- Row Header. -->
   <RowHeader {expanded} rowId={id} title={name} {rowDragMoveDisabled} on:mouseDownRowMove on:toggleRowExpansion>
-    <div slot="right">
+    <div slot="right" class="row-controls">
```

Comment:
Thoughts on showing this only where there are > 1 activity layers?

Reply:
Do you think that would potentially be confusing if it conditionally shows up? I know that some users like consistent icons, but if you think that's not needed for this I'm ok with making show up for more than 1

Reply:
Oh it's more that if you don't have any activity layers the button isn't clickable so might be more confusing in that sense?

Reply:
Oooh. I see what you mean. Yeah I think that makes sense. Should the same thing be applied to the "toggle all" button?

Reply:
Ok. I think I have it behaving the way you were suggesting. Thanks!